nsv13/code/game/objects/items/munitions_items.dm

```diff
@@ -70,6 +75,12 @@
 	icon_state = "sec_headset"
 	keyslot = new /obj/item/encryptionkey/munitions_tech
 
+/obj/item/radio/headset/headset_bridge

```

Comment:
Not a munitions item.

Reply:
Bridge Staff is under the munitions header in the Job Select screen, hence...

Reply:
It is under the Job Select screen to avoid certain issues. Hence not.

Reply:
So am I to place this in a separate file and add it to the #includes?

Reply:
Yes a separate file and add it to the .dme